### ISSUE TITLE
A11y issues

### ISSUE THREAD

**Issue by finnp** [jsconf/2015.jsconf.eu#10]
There are some violations of the WCAG standard: http://wave.webaim.org/report#/2015.jsconf.eu

I might PR something soon, but wanted to let you know in case I don't find the time for it. I also shared some tips for this here: https://github.com/finnp/a11y-talk

Best,
Finn
